src/exception_frame.rs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use phper::sys;
+
+pub struct ExceptionFrame {
+    #[allow(dead_code)]
+    not_matter: i32,
```

Comment:
This code is not necessary, so remove it?

Reply:
I'm not that familiar with Rust patterns. Since we are applying an [RAII](https://en.wikipedia.org/wiki/Resource_acquisition_is_initialization) design here, the dead code could prevent a client from accidentally bypassing the `new` operation.

Reply:
Maybe you can use [PhantomData](https://doc.rust-lang.org/std/marker/struct.PhantomData.html) here.